### PR TITLE
refactor: move separator into slice from related articles

### PR DESCRIPTION
**Pull request by colincclark**
Move the separators for Slice child components to the Slice package as discussed

### CODE REVIEW

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/861f02e56ad8df0dc8311e2afcd0722b1475d8b0/dextrose-test/index.html

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/cb4235062f49c0dd66a45b248353d1c7e941b6e0/dextrose-test/index.html

File: packages/slice/styles/default/responsive.web.js [hunk 1]
```diff
@@ -25,16 +16,18 @@ export const ChildrenContainerStyles = childCount => ({
...
     return `
       flex-direction: row;
       margin: 0 auto;
-      ${childCount >= 5 ? largeStyle : smallStyle}
+      ${childCount === 3 ? largeStyle : smallStyle}
     `;
   },
   wideUp: () => `
-    width: ${childCount >= 5 ? "auto" : config.wideBpWidth};
+    width: ${childCount === 3 ? "100%" : config.wideBpWidth};
```
Comment (nikhedonia) [inline packages/slice/styles/default/responsive.web.js]: 👍 that makes more sense

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/285b957b28c4bfe02ee461f58b961c37676ff206/dextrose-test/index.html

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/951647f52b47a0f59fb20d9e93aa1058f2ab774b/dextrose-test/index.html

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/951647f52b47a0f59fb20d9e93aa1058f2ab774b/dextrose-test/index.html

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/97326c2c07d149775e5d2a65bfd3b7edc7375d9c/dextrose-test/index.html

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/a0ece89be44a6b53ae7a18aa1a90422eacde1a39/dextrose-test/index.html

File: packages/slice/__tests__/shared.js [hunk 2]
```diff
@@ -2,34 +2,50 @@
...
 import React from "react";
-import renderer from "react-test-renderer";
+import { shallow } from "enzyme";
 import Slice from "../slice";
 
 module.exports = () => {
   context("DEFAULT template", () => {
     const template = "DEFAULT";
 
+    const items = [
```
Comment (jimhunty) [inline packages/slice/__tests__/shared.js]: Surely just an array is necessary here?

File: packages/slice/slice.stories.js [hunk 3]
```diff
@@ -1,21 +1,36 @@
...
-  flex: 1,
+const moduleStyle = StyleSheet.create({
   height: 150
-};
+});
 
-const Module = backgroundColor => (
-  <View style={[moduleStyle, backgroundColor]} />
-);
+const colours = [
```
Comment (jimhunty) [inline packages/slice/slice.stories.js]: Can it just be an array and use a modified index value for key?

**Pull request merged**